platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/platformstatus/PlatformStatus.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (C) 2022-2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.common.system.platformstatus;
+
+import com.swirlds.common.UniqueId;
+
+/**
+ * The status of the Platform
+ * <p>
+ * NOTE: not all of these statuses can currently be reached. The status state machine is still under development. In
+ * such cases, the documentation will indicate that the status is "not in use"
+ *
+ */
+public enum PlatformStatus implements UniqueId {
```

Comment:
nitpick, some javadocs end with a `.`, but others do not. My personal preference in javadocs is to end sentences with `.` if writing full sentences.